packages/devtools_app/lib/src/screens/vm_developer/object_inspector_view.dart

```diff
@@ -46,18 +52,59 @@ class _ObjectInspectorViewState extends State<_ObjectInspectorView> {
         Provider.of<VMDeveloperToolsController>(context);
     controller = vmDeveloperToolsController.objectInspectorViewController!
       ..init();
+    tabController = TabController(
+      length: 2,
+      vsync: this,
+      // Disable the animation.
+      animationDuration: const Duration(),
```

Comment:
what's the reasoning for disabling this?

Reply:
It just looks really out of place and makes navigating between the program explorer and object store look slow.